### ISSUE TITLE
python binaries broken inside third_party

### ISSUE THREAD

**Issue by matthen** [thought-machine/please#451]
If I have in my config 

```
[python]
moduledir = "third_party.python"
```

and add a python_binary anywhere inside `third_party/python`, then running it fails with the message:

```
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.6.4_4/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.6.4_4/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "plz-out/bin/third_party/python/binary.pex/__main__.py", line 238, in <module>
  File "plz-out/bin/third_party/python/binary.pex/__main__.py", line 230, in run
  File "plz-out/bin/third_party/python/binary.pex/__main__.py", line 203, in interact
  File "plz-out/bin/third_party/python/binary.pex/__main__.py", line 212, in main
  File "/usr/local/Cellar/python/3.6.4_4/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 201, in run_module
    mod_name, mod_spec, code = _get_module_details(mod_name)
  File "/usr/local/Cellar/python/3.6.4_4/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 153, in _get_module_details
    code = loader.get_code(mod_name)
AttributeError: 'ModuleDirImport' object has no attribute 'get_code'
```

moving the binary outside of third party, or removing moduledir from the config fixes it.
this is in 13.2.1

**Comment by peterebden** [comment]
Oh interesting. I suppose the simple answer is "put the binary somewhere else", but I don't imagine it will be anything fundamental. Luna, do you mind having a look please?

**Comment by matthen** [comment]
Cool, yeah I moved it elsewhere but thought it was a bug worth reporting

**Comment by BNMetrics** [comment]
This issue has been fixed in #458
